spec/features/view_peeps_spec.rb

```diff
@@ -0,0 +1,14 @@
+feature "viewing peeps" do
+require_relative './helpers/sign_up.rb'
```

Comment:
these are not indented correctly and I think they'd be better off in spec_helper.rb